linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/install_smartcard_packages/ansible/sle15.yml

```diff
@@ -0,0 +1,24 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = enable
+# complexity = low
+# disruption = low
+
+- name: Set smartcard packages fact
+  set_fact:
+    smartcard_packages:
+      - pam_pkcs11
+      - mozilla-nss
+      - mozilla-nss-tools
+      - pcsc-ccid
+      - pcsc-lite
+      - pcsc-tools
+      - opensc
+{{% if product not in ["sle15"] %}}
+      - coolkey
+{{% endif %}}
```

Comment:
If the file is named sle15.yml the situation that product is not in sle15 should never happen, right?

Reply:
Similar problem will be also in sle12.yml.

Reply:
that's true. I really don't like this solution. Do you know if we could solve this somehow differently?

Reply:
We can solve it by removing these 3 lines.

Reply:
I understand. I meant more a solution that would not require these duplicated files

Reply:
it's a bit hard to implement this change as there are multiple template files that need to be changed and some of them would not support the multiple packages requirement. Let's do the minor change required for this PR to get merged.

Reply:
done by push force